lib/screens/home/explorer/staking_modal/staking_initial.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:provenance_wallet/common/pw_design.dart';
+import 'package:provenance_wallet/common/widgets/button.dart';
+import 'package:provenance_wallet/common/widgets/pw_dropdown.dart';
+import 'package:provenance_wallet/common/widgets/pw_list_divider.dart';
+import 'package:provenance_wallet/screens/home/explorer/staking_modal/staking_modal_bloc.dart';
+import 'package:provenance_wallet/screens/home/transactions/details_item.dart';
+import 'package:provenance_wallet/util/get.dart';
+import 'package:provenance_wallet/util/strings.dart';
+
+class StakingInitial extends StatelessWidget {
+  const StakingInitial({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final bloc = get<StakingModalBloc>();
+
+    return StreamBuilder<StakingModalDetails>(
+      initialData: bloc.stakingModalDetails.value,
+      stream: bloc.stakingModalDetails,
+      builder: (context, snapshot) {
+        final details = snapshot.data;
+        if (details == null) {
+          return Container();
+        }
+        return ListView(
+          children: [
+            DetailsItem(
+              title: Strings.stakingInitialDescription,
+              endChild: Flexible(
+                child: PwText(
+                  details.validator.description,
+                  overflow: TextOverflow.ellipsis,
```

Comment:
Verify that the ellipsis are baseline aligned with the text instead of vertically centered. If not, might need to use a different overflow or fix the ellipsis bug.

Reply:
..they aren't. Fade looks better anyway.